### PR TITLE
gui: fix status widget flashing with stopped suite

### LINKED ISSUE #2035: minor gcylc GUI issue, for stopped suites
When gcylc is attempting to connect to a stopped suite, the status widget almost imperceptibly flashes from (e.g.) `stopped with succeeded` to just `stopped` when the connection timer reaches zero.  This creates a quick but annoying flicker in the status bar.

### PR DESCRIPTION
**Pull request by matthewrmshin**
Should fix #2035.

@hjoliver please let me know if this fixes the issue or not.

### CODE REVIEW

File: lib/cylc/gui/updater.py [hunk 1]
```diff
@@ -205,15 +205,21 @@ def reconnect(self):
...
+                    self.status = SUITE_STATUS_STOPPED
+                    gobject.idle_add(
+                        self.info_bar.set_stop_summary, stop_summary)
+            try:
+                update_time_str = get_time_string_from_unix_time(
+                    self.stop_summary[0]["last_updated"])
+            except (AttributeError, IndexError, KeyError):
+                update_time_str = None
+            self.info_bar.set_update_time(
+                update_time_str, self.info_bar.DISCONNECTED_TEXT)
```
Comment (hjoliver) [inline lib/cylc/gui/updater.py]: This call should be made via `gobject.idle_add` too?

Comment (matthewrmshin) [inline lib/cylc/gui/updater.py]: Done.

**Comment by matthewrmshin** [comment]
@oliver-sanders please review 2.

**Comment by oliver-sanders** [comment]
Looks OK, fixes the annoying flickering.

**Pull request merged**